_articles/uploading_in_bulk.md

```diff
@@ -59,7 +59,7 @@ Files can be downloaded in bulk using the `syncFromSynapse` function. This funct
 
 ## Editing in Bulk
 
-You can edit files in bulk by changing the values in the manifest and pushing it up to Synapse using the `syncToSynapse` function. The manifest allows you to modify everything: file path, provenance, annotations, and versions. However, if only annotations are being updated, we recommend using our [File Views]({{ site.baseurl }}{% link _articles/views.md %}) feature.
+You can edit files in bulk by changing the values in the manifest and pushing it up to Synapse using the `syncToSynapse` function. The manifest allows you to modify everything: file path, provenance, annotations, and versions. If you only want to update the file annotations, we recommend using our [File Views]({{ site.baseurl }}{% link _articles/views.md %}) feature. However, updating annotations is still possible using the manifest. If only updating annotations and the files have not changed, adding the manifest column `forceVersion` with the value of `False` for each row will stop `syncToSynapse` from uploading new versions of the files.
```

Comment:
```suggestion
You can modify values in the manifest and re-upload it to Synapse using `syncToSynapse` to edit files in bulk. The manifest allows you to modify everything: file path, provenance, annotations, and versions. If the files have not changed and you only want to update the file annotations, add a column called **forceVersion** to the manifest with the value **False** for each row. This will stop `syncToSynapse` from uploading new versions of the files.

You can also update annotations using [File Views]({{ site.baseurl }}{% link _articles/views.md %}).
```

Reply:
I changed the order so the content is consistently about the manifest before it dives into a file view.

Do you think it makes sense to update the entire article to "Upload and Download Data in Bulk using a manifest" and then separately have an article "Upload and Download Data in Bulk using File Views"? Currently, from Views you can navigate to Download Options > Programmatic Options to get a query for downloading data. That would be a new issue.

Reply:
I like that ordering better, Kelsey. Thanks! I was trying to not alter the original content too much, but I forget that we are updating these docs to be better anyway. 

One question I have is if bold is how we are addressing values in the rest of the docs. I haven't noticed this before.

As for adding upload/download in bulk using File Views, I don't think that would be appropriate for upload, would it? You can edit annotations in bulk that way, but I don't know about uploading files that way. It would be nice to document downloading them, though. At the very least, if that isn't covered in the File Views section, it should definitely be covered somewhere. Maybe "Download and Update in Bulk using File Views"?

Reply:
You are correct - there is no upload functionality in File Views for files itself, rather annotations/metadata can be "uploaded" with file views. I will open an issue and maybe next week we can discuss how to frame it.

Our [style guide](https://github.com/sage-bionetworks/synapsedocs#style-guide) does not explicitly call out how to deal with values. We recently updated the convention for code formatting to be used in entities or features when they are first mentioned and of course code itself. We found the formatted code was distracting and wanted to move towards a model where it is used less. You will see it is still very much present throughout the copy...

Reply:
Great! Thanks!